Libraries/LibLine/Editor.cpp

```diff
@@ -559,14 +562,11 @@ void Editor::handle_read_event()
             }
             case 'D': // left
 		do_cursor_left(ctrl_held ? Word : Character);
-                m_inline_search_cursor = m_cursor;
```

Comment:
So this would always search for the entire phrase now?
I guess we can just delete from the end to get the inline search phrase to be right, but...couldn't we have made it so C-{b,f} update this too?
```sh
> f|oo
   ^ cursor here, search phrase is "f"
> fo|o
    ^ cursor here, search phrase is "fo"
```

Reply:
No, I just moved this into do_cursor_left(), so that it also happens for ctrl-b.

Reply:
Ah. understood.
the commit message could perhaps be a bit less..."remove"y and more "move"y?

Reply:
I was certain I had written "Move...". Not sure how I didn't. Fixed now, thanks for catching :)